package.json

```diff
@@ -27,7 +27,7 @@
     "parser": "@babel/eslint-parser"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.17.0",
+    "@babel/eslint-parser": "^7.18.9",
```

Comment:
Another unrelated change

Reply:
Sorry new to git. Might be due to accidentally updated the @bael/eslint-parser...